### ISSUE TITLE
Not working with Thunderbird 108.x

### ISSUE THREAD

**Issue by berlincount** [lkosson/full-address-column#12]
Maybe just needs a trivial update?

Thanks for this great piece of software!

**Comment by lkosson** [comment]
Yep, it just needed `strict_max_version` bump to `108` to work on beta. Thunderbird add-on gallery won't accept it for now, though.

**Comment by BoomerangAide** [comment]
Not working on 102.9.1 (64 bits).
Is it the same issue?

**Comment by berlincount** [comment]
Hm. It installs now (1.0.4 on 114.0b4 (32-bit)), but doesn't offer the columns in the selection 🤔

**Comment by lkosson** [comment]
New version needs Thunderbird 115.10+. Old version still works in Thunderbird 102.
